src/Tutor.Core/LearnerModel/DomainOverlay/KnowledgeComponentMasteries/Events/KnowledgeComponentEvent.cs

```diff
@@ -1,10 +1,13 @@
-﻿using Tutor.Core.BuildingBlocks.EventSourcing;
+﻿using System;
+using Tutor.Core.BuildingBlocks.EventSourcing;
 
 namespace Tutor.Core.LearnerModel.DomainOverlay.KnowledgeComponentMasteries.Events
 {
     public abstract class KnowledgeComponentEvent : DomainEvent
     {
         public int KnowledgeComponentId { get; set; }
         public int LearnerId { get; set; }
+        public TimeSpan DurationOfPreviousSessions { get; set; }
+        public TimeSpan DurationOfCurrentSession { get; set; }
```

Comment:
For now I just added these to all KC events as an example, I wasn't sure what to include in which events from the data we now have in SessionTracker. 

Also, the TimeSpan class is serialized like this by default:
![image](https://user-images.githubusercontent.com/57844740/190917686-b7870a2f-0c6d-400e-bdc8-de76605fc900.png)
So that should probably be cleaned up if we keep TimeSpan properties in events, but I'm not quite sure what the easiest way to do that is.

Reply:
Event properties adjusted in e27e7e9274f6a6072fed45df0051c37a95703439.